src/shared/common/components/workstack.jsx

```diff
@@ -225,4 +255,8 @@ const WrappedWorkstackAllocate = props => (
     </ApplicationConsumer>
 );
 
+const renderHeader = (column) => {
```

Comment:
this might be neater before the component wrapping.  The other methods for this class are static methods on the class but this is not essential.

Reply:
moved to where the rest of the methods are declared